sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/VirtualMachine.java

```diff
@@ -2539,6 +2539,24 @@ interface WithSecurityFeatures {
             Update withoutVTpm();
         }
 
+        /** The stage of the definition allowing to specify the security profile for the virtual machine. */
+        interface WithSecurityProfile {
+
+            /**
+             * Request to enable the Host Encryption for the virtual machine.
```

Comment:
Just Enables/Disabled in the Javadoc. No need for the "Request to".

Same to the create flow (if the contributor uses "Request to").

Reply:
Fixed in the new version.

Reply:
Enables , plural

Reply:
I mean the Javadoc